consensus/polybft/contractsapi/artifacts-gen/main.go

```diff
@@ -191,22 +189,32 @@ func main() {
 		},
 	}
 
+	str := `// This is auto-generated file. DO NOT EDIT.
+package contractsapi
+
+`
+
 	for _, v := range readContracts {
 		artifactBytes, err := artifact.ReadArtifactData(scpath, v.Path, getContractName(v.Path))
 		if err != nil {
 			log.Fatal(err)
 		}
 
-		f.Var().Id(v.Name + "Artifact").String().Op("=").Lit(string(artifactBytes))
+		dst := &bytes.Buffer{}
+		if err = json.Compact(dst, artifactBytes); err != nil {
+			log.Fatal(err)
+		}
+
+		str += fmt.Sprintf("var %sArtifact string = `%s`\n", v.Name, dst.String())
 	}
 
-	fl, err := os.Create(currentPath + "/../gen_sc_data.go")
+	output, err := format.Source([]byte(str))
 	if err != nil {
+		fmt.Println(str)
 		log.Fatal(err)
 	}
 
-	_, err = fmt.Fprintf(fl, "%#v", f)
-	if err != nil {
+	if err = os.WriteFile(currentPath+"/../gen_sc_data.go", output, 0600); err != nil {
```

Comment:
WDYT about using `common.SaveFileSafe` function?

```suggestion
	if err = common.SaveFileSafe(currentPath+"/../gen_sc_data.go", output, 0600); err != nil {
```

Reply:
Done